LogsService.RabbitMQ.Tests/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using RabbitMQ.Client;
+
+namespace LogsService.RabbitMQ.Tests
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var count = 0;
+
+            do
+            {
+                var time = new Random().Next(1000, 3000);
```

Comment:
Why you use random intervals instead of constant value?

Reply:
I want emulate random events in RabbitMq and floating load on the RabbitMq server